lib/pbench/server/database/alembic.ini

```diff
@@ -80,6 +80,12 @@ args = (sys.stderr,)
 level = NOTSET
 formatter = generic
 
+[handler_fileHandler]
+class=FileHandler
```

Comment:
Should we be using SysLogHandler instead of logging to a private/obscure file?

Reply:
Fixed. I was also trying to experiment with this package [systemd-python](https://github.com/systemd/python-systemd). it is a python extension for systemd and helps us log to journald. We need to add a handler like 
```
from systemd import journal
logger.addHandler(journal.JournalHandler())
```
However, this adds an extra requirement. Therefore decided to go with `SyslogHandler` since I found out that logs written via `/dev/log` are also accessible via journald